nightfall-deployer/src/contract-setup.mjs

```diff
@@ -4,96 +4,62 @@ address of the contract that holds global state (State.sol)
 */
 
 import config from 'config';
-import logger from 'common-files/utils/logger.mjs';
 import Web3 from 'common-files/utils/web3.mjs';
 import { waitForContract } from 'common-files/utils/contract.mjs';
+import logger from 'common-files/utils/logger.mjs';
 
 async function setupContracts() {
+  const web3 = Web3.connection();
+
   const stateInstance = await waitForContract('State');
   logger.debug(`address of State contract is ${stateInstance.options.address}`);
 
-  const simpleMultiSigAddress = (await waitForContract('SimpleMultiSig')).options.address;
-  const shieldContractInstance = await waitForContract('Shield');
-  const stateContractInstance = await waitForContract('State');
-  const proposerContractInstance = await waitForContract('Proposers');
-  const challengesContractInstance = await waitForContract('Challenges');
+  const tx = {
+    from: process.env.FROM_ADDRESS,
```

Comment:
Why are we mixing `process.env` variables with `config` ones here? It would be better to put all in `config` variables that check this `process.env.FROM_ADDRESS`.

Reply:
This `FROM_ADDRESS` is needed as environment variable, actually we do the same for `geth` deployments. We could do something like this in `config`:

`
FROM_ADDRESS: process.env.FROM_ADDRESS || 0x.....
`

And then use `config.FROM_ADDRESS` here. But I don't see the benefit, and that's why I just left it as it is.

I agree our codebase is very dependent on environment variables, we should think of a better way